src/rag_processor/api/user.py

```
@@ -0,0 +1,51 @@
+    user_id: str | None = None
+    groups: list[str] = []
+
+
+@router.get(
+    "/me",
+    response_model=UserResponse,
+    summary="Get current user",
+    description="Returns the currently authenticated user's information.",
+)
+async def get_me(
+    user: CloudflareUser = Depends(get_current_user),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)